packages/pharos/src/components/modal/pharos-modal.ts

```diff
@@ -93,7 +93,7 @@ export class PharosModal extends LitElement {
   }
 
   protected updated(changedProperties: PropertyValues): void {
-    if (changedProperties.has('open')) {
+    if (changedProperties.has('open') && changedProperties.get('open') !== undefined) {
```

Comment:
Think we only want to wrap this check around where `_emitVisibilityChange` gets called. Believe this will address the regressions from Chromatic.

Reply:
ah gotcha done!